### ISSUE TITLE
github username

### ISSUE THREAD

**Issue by H1D** [contrib/contrib#11]
When I try do submit a hotfix `contrib` ask my Github credentials. And seems like it cannot recognize my username as legit github username:
![image](https://cloud.githubusercontent.com/assets/697625/5835651/1e167ab6-a17c-11e4-92f7-6da9e8457f4c.png)

BTW I have my public key copied to GH account, so it is strange that `contrib` ask for credentials
Maybe it is not contrib related thing, then redirect me.

**Comment by heff** [comment]
Oh noes! That's a poor verification on the username. And it looks like it's git verifying, not contrib. I'm not sure what would need to be done to fix that verification specifically.

So far contrib doesn't ever ask for your github password directly (at least that I can remember), but the underlying command line scripts that it runs may.

@mmcc, is this the issue were talking about yesterday? I believe it was that the contrib.json assume the https protocol for the repo. I wonder if you install the project using `contrib install videojs/video.js` if you'd run into the same issue.

**Comment by mmcc** [comment]
@heff nah, the issue I was running into turned out to be a me problem, not contrib. Contrib just uses the remotes specified, and I had specified an HTTPS url, so that's why I was running into issues.
